### PR TITLE
WPF/System.Windows.Controls.Ribbon:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool

### PR DESCRIPTION
**Pull request by ryalanms**
WPF/System.Windows.Controls.Ribbon:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool.

The RC1 source for these assemblies has 'triple-slash' comments not in the documentation:

*PresentationBuildTasks
PresentationCore
ReachFramework
System.Windows.Controls.Ribbon
System.Windows.Presentation
UIAutomationClient
UIAutomationTypes
WindowsBase*

There is one documentation update PR per assembly. 

(Hide whitespace changes when reviewing files.) 

/cc @dotnet/wpf-developers @carlossanlop

### CODE REVIEW

File: xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml [hunk 1]
```diff
@@ -1,173 +1,175 @@
...
+        <returns>The converted value.</returns>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="value" /> is <see langword="null" />.  
+  
+ -or-  
+  
+ <paramref name="destinationType" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.NotSupportedException">A NotSupportedException is thrown if the example object is null
+            or if the destinationType isn't one of the valid destination types.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml:171]: This exception contradicts the already existing `ArgumentNullException` above.

File: xml/System.Windows.Controls.Ribbon/RibbonControlLengthUnitType.xml [hunk 2]
```diff
@@ -1,111 +1,120 @@
...
+    <BaseTypeName>System.Enum</BaseTypeName>
+  </Base>
+  <Docs>
+    <summary>Defines constants that specify how elements in a <see cref="T:System.Windows.Controls.Ribbon.Ribbon" /> are sized.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Note: Keep the RibbonControlLengthUnitType enum in sync with the string representation
```
Comment (carlossanlop) [inline xml/System.Windows.Controls.Ribbon/RibbonControlLengthUnitType.xml:24]: Are these remarks relevant to the final user?

File: xml/System.Windows.Controls.Ribbon/RibbonGallery.xml [hunk 3]
```diff
@@ -1,2974 +1,2991 @@
...
+      <ReturnValue>
+        <ReturnType>System.Windows.DependencyProperty</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Identifies the <see cref="P:System.Windows.Controls.Ribbon.RibbonGallery.FilterItemTemplate" /> dependency property.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="FilterItemTemplateSelector">
+      <MemberSignature Language="C#" Value
```
Comment (carlossanlop) [inline xml/System.Windows.Controls.Ribbon/RibbonGallery.xml:2271]:
```suggestion
This method currently does not support the virtualization.

```

Comment (carlossanlop) [inline xml/System.Windows.Controls.Ribbon/RibbonGallery.xml:1440]: Is this relevant to the final user, or is it directed to maintainers?

File: xml/System.Windows.Controls.Ribbon/RibbonTextBox.xml [hunk 4]
```diff
@@ -1,1939 +1,1948 @@
...
+        <summary>Gets a value that indicates whether the control is enabled.</summary>
+        <value>
+          <see langword="true" /> if the control is enabled, otherwise <see langword="false" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The reason this property is overridden is so that RibbonTextBox
+                can infuse the value for CanExecute into it.
```
Comment (carlossanlop) [inline xml/System.Windows.Controls.Ribbon/RibbonTextBox.xml:499]: Relevant to final users?

File: xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml:171]: Maybe this is a case of a breaking change between .NET Framework and .NET Core? In which case, the difference should be explicitly mentioned.

File: xml/System.Windows.Controls.Ribbon/RibbonGallery.xml [hunk 5]
```diff
@@ -1,2974 +1,2992 @@
...
```
Comment (gewarren) [inline xml/System.Windows.Controls.Ribbon/RibbonGallery.xml:1439]: Probably not relevant?

Comment (gewarren) [inline xml/System.Windows.Controls.Ribbon/RibbonGallery.xml:2271]: This isn't that clear to me.

File: xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml (hunk 1 above)
Comment (ryalanms) [reply xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml:171]: This looks correct.  ArgumentNullException when Value parameter is null and NotSupportedException when destinationType is null.

**Review by gewarren — APPROVED** [review]
